application/backend/src/main/resources/apps/typerefinery/components/widgets/smallticker/clientlibs/functions.js

```diff
@@ -0,0 +1,112 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Components.Widgets = Typerefinery.Components.Widgets || {};
+window.Typerefinery.Components.Widgets.smallTicker =
```

Comment:
Please use proper namespaces.

Using caps in some places and small letters in some places, I think I have already shared this.

Reply:
Done